readme.md

```diff
@@ -221,6 +228,33 @@ You can optionally pass one or more of these parameters every time you use Lambo
   ```bash
   lambo new superApplication --full
 
+## GitHub Repository Creation
+
+- `--github=<options>` to create a new GitHub repository and push your new project to it.
+
+  ```bash
+  lambo new superApplication --github='--private'
+  ```
+
+Lambo uses the official GitHub command line tool to create your new repository so you must have it installed. You can find installation instructions [here](https://github.com/cli/cli#installation).
+
+New repository creation is configured by passing the value of the Lambo `--github` command line option directly to the GitHub command line tool. You must pass at least the desired visibility of your new repository, one of, `--private`, `--public`, or `--internal`.
+
+ ```bash
+ # Create a new GitHub repository <your_github_username>/superApplication with the desired visibility
```

Comment:
ahhh.. so does it default to my username unless I intentionally specify the org? That makes sense! that way I can say `lambo new SuperApplication` and it will push to github.com/mattstauffer/SuperApplication`.

However, what do I do if I want to push to `tighten/SuperApplication`? Do I have to do the whole `--org=tighten` or something? I wonder if there's a better syntax?

And I wonder.. instead of pushing to `SuperApplication`, would it be worth considering snake-casing it so it instead pushes to `super-application`?

Reply:
> ahhh.. so does it default to my username unless I intentionally specify the org? That makes sense! that way I can say lambo new SuperApplication and it will push to github.com/mattstauffer/SuperApplication`.

Correct.

> However, what do I do if I want to push to tighten/SuperApplication? Do I have to do the whole --org=tighten or something? I wonder if there's a better syntax?

Yes, you need to use `--org=tighten`. `gh repo create my-app` will create a repo in your account (mattstauffer) and `gh repo create tighten/my-app` will create tighten/my-app assuming you have access rights to that org. Because the gh command doesent have an explicit org flag we have to create it with interpolation/concatenation. Thats why I have a separate flag. It's just easier that way.

I also choose to mirror the Laravel installer api.

> And I wonder... instead of pushing to SuperApplication, would it be worth considering snake-casing it so it instead pushes to super-application?

I did think about that and I know snake-case is the commonly accepted naming style but then I thought should we force that? I figured let people do what they want.